docs/model-reuse.md

```
@@ -243,10 +244,98 @@ models/
+```
+
+**Step 2: Generate with `--use-type-alias`**
+
+```bash
+datamodel-codegen \
+  --input schema.json \
+  --output model.py \
+  --use-type-alias
+```
+
+**Result: Single TypeAlias reused across classes**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)